docs/docs.md

```diff
@@ -168,11 +171,22 @@ The [`config/log.rego`](../config/log.rego) file ensures that sensitive informat
 
 ## Component 3: the integrator authentication endpoint
 
-The nodejs server provides an [`/authenticated`](../example-app/index.js#L56) endpoint which can be used to check if the caller is logged in.
+The nodejs server provides an [`/jsonwebtoken`](../example-app/index.js#L57) endpoint which can be used to generate a signed JSON Web Token authorizing the request.
```

Comment:
```suggestion
The nodejs server provides a [`/jsonwebtoken`](../example-app/index.js#L57) endpoint which can be used to generate a signed JSON Web Token authorizing the request.
```

Pronoun copy-edit: the following word now begins with a consonant sound, so the pronoun loses the final _n_ character.